modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseServiceWaitStrategy.java

```diff
@@ -19,18 +19,21 @@
  * Created on 06/06/2017
  */
 @Slf4j
-public class CouchbaseQueryServiceWaitStrategy extends AbstractWaitStrategy {
+public class CouchbaseServiceWaitStrategy extends AbstractWaitStrategy {
```

Comment:
since this class is public, this change will break the binary compatibility. Consider extracting a base (internal) class instead